README.md

```diff
@@ -3,8 +3,8 @@
 [![Limitador GH Workflow](https://github.com/Kuadrant/limitador/actions/workflows/rust.yml/badge.svg)](https://github.com/Kuadrant/limitador/actions/workflows/rust.yml)
 [![docs.rs](https://docs.rs/limitador/badge.svg)](https://docs.rs/limitador)
 [![Crates.io](https://img.shields.io/crates/v/limitador)](https://crates.io/crates/limitador)
-[![Docker Repository on Quay](https://quay.io/repository/3scale/limitador/status
```

Comment:
this badge only makes sense if the image is being build using quay.io building system. Currently it is being build using GH actions, so this badge should point to GH workflows [status badge](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge)